### PR TITLE
Fix flakiness of test_timeline_copy.

### PR DESCRIPTION
**Pull request by arssher**
Timeline might be not initialized when timeline_start_lsn is queried. Spotted by CI.

### CODE REVIEW

File: test_runner/fixtures/safekeeper/http.py [hunk 1]
```diff
@@ -161,6 +162,16 @@ def timeline_status(
...
 
+    # Get timeline_start_lsn, waiting until it's nonzero. It is a way to ensure
+    # that the timeline is fully initialized at the safekeeper.
+    def get_non_zero_timeline_start_lsn(self, tenant_id: TenantId, timeline_id: TimelineId) -> Lsn:
+        def timeline_start_lsn_non_zero():
+            s = self.timeline_status(tenant_id, timeline_id).timeline_start_lsn
+            assert s > Lsn(0)
+
+        wait_until(30, 1, timeline_start_lsn_non_zero)
+        return self.timeline_status(tenant_id, timeline_id).timeline_start_lsn
```
Comment (koivunej) [inline test_runner/fixtures/safekeeper/http.py:173]:
```suggestion
            assert s > Lsn(0)

        wait_until(30, 1, timeline_start_lsn_non_zero)
        return self.timeline_status(tenant_id, timeline_id).timeline_start_lsn
```
```suggestion
            assert s > Lsn(0)
            return s

        return wait_until(30, 1, timeline_start_lsn_non_zero)
```

**Review by koivunej — APPROVED** [review]

File: test_runner/fixtures/safekeeper/http.py (hunk 1 above)
Comment (arssher) [reply test_runner/fixtures/safekeeper/http.py]: applied

Comment (koivunej) [reply test_runner/fixtures/safekeeper/http.py]: aah yes sorry I forgot the return type. always something with python :)

File: test_runner/regress/test_wal_acceptor.py [hunk 2]
```diff
@@ -2083,8 +2083,13 @@ def remember_lsn():
 
     endpoint.safe_psql("create table t(key int, value text)")
 
-    timeline_status = env.safekeepers[0].http_client().timeline_status(tenant_id, timeline_id)
-    timeline_start_lsn = timeline_status.timeline_start_lsn
+    # Note: currently timelines on sks are created by compute and commit of
```
Comment (hlinnaka) [inline test_runner/regress/test_wal_acceptor.py:2086]: > ... timelines on sks are created by compute ...

What exactly does that mean? The compute doesn't make an explicit "create timeline" call to the safekeepers

Comment (arssher) [reply test_runner/regress/test_wal_acceptor.py:2086]: It is created on first connection from walproposer (which eventually will be fixed).

**Pull request merged**